pkg/pxc/app/statefulset/haproxy.go

```diff
@@ -106,7 +96,18 @@ func (c *HAProxy) AppContainer(spec *api.PodSpec, secrets string, cr *api.Percon
 		SecurityContext: spec.ContainerSecurityContext,
 	}
 
+	redinessDelay := int32(15)
+	if spec.ReadinessInitialDelaySeconds != nil {
```

Comment:
better to move this part to the if below

Reply:
done